scripts/helper.py

```diff
@@ -137,7 +134,7 @@ def get_markdown_table(namespace, title_prefix="##", link=False):
     # Replaces one newlines with two as otherwise double newlines do not show up in markdown
     output += namespace["description"].replace("\n", "\n\n") + "\n"
 
-    titles = ["Field", "Description", "Type", "Multi Field", "Example"]
+    titles = ["Field", "Level", "Description", "Type", "Multi Field", "Example"]
```

Comment:
I would put the level after the Type.

As we don't have multi fields anymore, we can remove the multi field part? Or should we keep it just in case. Not related to this PR but stumbled over it.

Reply:
I agree level at column 2 is not ideal.

However putting level after type puts it between type and multi-field, which are closely related. I'll try with level just before type instead, and let's see how this looks.